src/app/products/loan-products/edit-loan-product/edit-loan-product.component.ts

```diff
@@ -115,6 +136,10 @@ export class EditLoanProductComponent implements OnInit {
     this.wasPaymentAllocationChanged = true;
   }
 
+  setSupportedInterestRefundTypes(supportedInterestRefundTypes: StringEnumOptionData[]): void {
```

Comment:
Use the `OptionData` already defined in 'app/shared/models/option-data.model'

Reply:
Its not the same. OptionData id is number. StringEnumOptionData id is string